src/tools/rest-backend/config.cpp

```diff
@@ -0,0 +1,152 @@
+/**
+ * @file
+ *
+ * @brief configuration helper functions
+ *
+ * @copyright BSD License (see doc/LICENSE.md or http://www.libelektra.org)
+ */
+
+#include <iostream>
+#include <string>
+
+#include <config.hpp>
+
+namespace kdbrest
+{
+
+/* set dummy variables for statics */
+
+// JWT settings
+std::string Config::jwt_encryption_key = std::string ();
```

Comment:
You are basically right, I could also use the json::value. But at some point I need to validate the configuration anyway (on startup).

But it doesn't seem perfect either way; cppcms tools have weird behavior here and there. The worst is that you can use `json::value::set('this.are.objects.with.a.variable', true)` to set nested objects, but you can't use the same for arrays. I tried:
```
json::value::set('gimme.array[0]', true)
json::value::set('gimme.array.[0]', true)
json::value::set('gimme.array.0', true)
json::value::set('gimme.array.#0', true)
```
If any of these did work, the whole transformation would be so much easier...